src/main/webapp/js/components/interface/query/query.less

```diff
@@ -497,3 +497,7 @@
 #querybuilder .standard-row {
 	height: 130px;
 }
+
+#querybuilder .griddle-container{
+    background-color:rgba(65, 58, 58, 0.3);
```

Comment:
@Robbie1977 Colours should come from the theme. We can adjust the transparency of one colour from the theme like this e.g. `background: fade(@background_color_widget, 90%)`

Reply:
updated